src/uct/ib/mlx5/dc/dc_mlx5.h

```diff
@@ -284,7 +285,7 @@ typedef struct {
 } uct_dc_mlx5_dci_pool_t;
 
 
-UCS_ARRAY_DECLARE_TYPE(uct_dc_dci_array_t, uint16_t, uct_dc_dci_t);
+UCS_ARRAY_DECLARE_TYPE(uct_dc_dci_array_t, uct_dci_index_t, uct_dc_dci_t);
```

Comment:
I don't see this change in the original PR: https://github.com/openucx/ucx/pull/10278/

Maybe need to open another one and fix it

Reply:
it was discovered during this PR, so indeed not in the original